mindmeld/converter/rasa.py

```diff
@@ -26,6 +26,9 @@
 
 logger = logging.getLogger(__name__)
 
+RASA_ENTITY_REGEX = re.compile(r"(\[(.*?)\]\((.*?)\))")
+MINDMELD_ENTITY_REGEX = re.compile(r"\{.*?\}")
```

Comment:
I think this would also match `some query with weird characters []()` and convert to `some query with weird characters {|}`

Is this desirable?

We could change it by using `.+` instead of `.*`